providers/google/tests/system/google/cloud/gcs/example_http_to_gcs.py

```diff
@@ -31,75 +31,51 @@
 else:
     # Airflow 2 path
     from airflow.decorators import task  # type: ignore[attr-defined,no-redef]
-from airflow.models.baseoperator import chain
 from airflow.models.dag import DAG
 from airflow.providers.google.cloud.operators.gcs import GCSCreateBucketOperator, GCSDeleteBucketOperator
 from airflow.providers.google.cloud.transfers.http_to_gcs import HttpToGCSOperator
-from airflow.providers.standard.operators.bash import BashOperator
 from airflow.utils.trigger_rule import TriggerRule
 
-from system.google import DEFAULT_GCP_SYSTEM_TEST_PROJECT_ID
 from tests_common.test_utils.api_client_helpers import create_airflow_connection, delete_airflow_connection
 
 ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID", "default")
-PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT") or DEFAULT_GCP_SYSTEM_TEST_PROJECT_ID
+PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT", "default")
 
 DAG_ID = "example_http_to_gcs"
 BUCKET_NAME = f"bucket-{DAG_ID}-{ENV_ID}"
 
-cmd = """
-#!/bin/bash
-
-echo 'foo' > /tmp/test_file
-
-cd /tmp
-
-nohup python3 -m http.server 8083 > /dev/null 2>&1 &
-
-echo $!
-sleep 2
-exit 0
-"""
 
 with DAG(
     DAG_ID,
     schedule="@once",
     start_date=datetime(2025, 1, 1),
     catchup=False,
-    tags=["example"],
+    tags=["example, http_to_gcs"],
```

Comment:
'tags' is already a list. Maybe we can put the "http_to_gcs" as a second item in the list